test/parallel/test-crypto-authenticated.js

```diff
@@ -93,7 +93,7 @@ for (var i in TEST_CASES) {
 
   (function() {
     if (!test.password) return;
-    if (common.hasFipsCrypto) {
+    if (crypto.hasFipsCrypto()) {
```

Comment:
Let's make `common.hasFipsCrypto` a getter instead, should save us from fixing all tests and make them compatible to older node.js versions.